core/arch/arm/include/kernel/ftrace.h

```diff
@@ -0,0 +1,42 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2019, Linaro Limited
+ */
+
+#ifndef __FTRACE_H
+#define __FTRACE_H
+
+#include <user_ta_header.h>
+#include <kernel/tee_ta_manager.h>
+
+/*
+ * ta_fbuf_init() - Initialize ta ftrace buffer
+ * @ta_head:	user ta header
```

Comment:
s/ta/TA/g  (here, below and at lines 17, 29, 30, 32, etc...)

Reply:
ok.